src/parser_urdf.cc

```diff
@@ -56,9 +56,9 @@ typedef std::map<std::string, std::vector<SDFExtensionPtr> >
 StringSDFExtensionPtrMap g_extensions;
 bool g_reduceFixedJoints;
 bool g_enforceLimits;
-const char kCollisionExt[] = "_collision";
-const char kVisualExt[] = "_visual";
-const char kLumpPrefix[] = "_fixed_joint_lump__";
+constexpr char kCollisionExt[] = "_collision";
```

Comment:
not sure if this is necessary, but okay.

Reply:
I didn't understand enough what `constexpr` does, you're right this is redundant. I've reverted it, https://github.com/gazebosim/sdformat/pull/1264/commits/53f017607b93819cea33b85fb5b8c04a436b432d